appveyor.yml

```diff
@@ -13,7 +13,7 @@ install:
   - cmd: SET PATH=C:\maven\apache-maven-3.2.5\bin;%JAVA_HOME%\bin;%PATH%
   - cmd: SET MAVEN_OPTS=-XX:MaxPermSize=2g -Xmx4g
   - cmd: SET JAVA_OPTS=-XX:MaxPermSize=2g -Xmx4g
-  - cmd: SET M2_HOME=C:\maven\apache-maven-3.2.5
+#  - cmd: SET M2_HOME=C:\maven\apache-maven-3.2.5
```

Comment:
I have never answer, but it was there before, and I just comment it. I don't know why It was there, but I think when I setup the windows CI I copied from somewhere.